lib/sshnp.dart

```diff
@@ -85,9 +83,10 @@ class SSHNP {
   @visibleForTesting
   late final String clientAtSign;
 
-  /// The username to use on the remote host in the ssh session. Is fetched
-  /// from the sshnpd by [fetchRemoteUserName] during [init]
-  late final String remoteUsername;
+  /// The username to use on the remote host in the ssh session. Either passed
+  /// through class constructor or  Is fetched from the sshnpd
```

Comment:
Please correct capitalization and spacing

Reply:
Done